witms/spiders/guardian.py

```diff
@@ -14,35 +14,28 @@ class GuardianSpider(Spider):
         restrict_css="a[data-link-name*=article]",
     )
 
-    @staticmethod
-    def get_news_article(response):
+    def parse(self, response):
         loader = ArticleLoader(item=Article(), response=response)
         loader.add_value("url", response.url)
         loader.add_value("portal", GuardianSpider.portal_name)
-        loader.add_xpath("section", '//meta[@property="article:section"]/@content')
-        loader.add_xpath("authors", '//meta[@name="author"]/@content')
-        loader.add_xpath("authors", '//a[@rel="author"]//text()')
-        loader.add_css("title", "h1 *::text")
-        loader.add_xpath("title", '//meta[@name="title"]/@content')
-        loader.add_xpath("title", '//meta[@property="og:title"]/@content')
-        loader.add_xpath("description", '//meta[@name="description"]/@content')
-        loader.add_xpath("description", '//meta[@property="og:description"]/@content')
+        loader.add_xpath("section", "//meta[@property=\"article:section\"]/@content")  # default
```

Comment:
Shouldn't the first of a kind not have that "default" comment? As in, the first selector tries to get a value, and the following ones fall back to a default value... That aside, I vote for removing those comments, they're repetitive and I don't feel like they add useful information.